### PR TITLE
Remove codacy badge from readme

### PR DESCRIPTION
**Pull request by khanguslee**
## Description

- Removes the codacy badge from the readme
- Removes information relating to V2 DAS.

## Screenshots

N/A

## Steps to Test

Read the `README.md` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
+
+#### DAS.js
 
-### DAS.js
 node.js program we run to communicate to the Teensy that is connected to the Raspberry Pi via serial communication.
 
-### Test/serial_test.py
+#### Test/serial_test.py
 
 Imitates the Teensy serial connection by creating a virtual serial port. Note that this will only work on Unix systems - if you're running Windows, perhaps try running the script under [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```
Comment (emilytrau) [inline README.md]: We could remove the teensy test scripts.

**Review by emilytrau — CHANGES_REQUESTED** [review]

**Review by emilytrau — DISMISSED** [review]
LGTM! We should remove the serial and Teensy test scripts from the Rapsi/Tests directory, but that can be another PR

**Review by hallgchris — DISMISSED** [review]

**Comment by khanguslee** [comment]
> LGTM! We should remove the serial and Teensy test scripts from the Rapsi/Tests directory, but that can be another PR

Thinking about creating a utils folder for our 'test' scripts

**Review by emilytrau — APPROVED** [review]

**Review by hallgchris — APPROVED** [review]

**Comment by khanguslee** [comment]
It's funny because we pulled in https://github.com/monash-human-power/data-acquisition-system/pull/49 when this PR was already here. oops. Closing this PR.

**Pull request closed without merging**